rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/ParserOnlyContribution.java

```diff
@@ -97,7 +102,8 @@ private static Either<IFunction, Exception> loadParser(ParserSpecification spec)
         try {
             logger.debug("Loading parser {} at {}", reifiedType, spec.getParserLocation());
             // this hides all the loading and instantiation details of Rascal-generated parsers
-            var parser = vf.loadParser(reifiedType, spec.getParserLocation(), VF.bool(spec.getAllowAmbiguity()), VF.bool(false), VF.bool(false), VF.bool(false), vf.set());
+            var parser = vf.loadParser(reifiedType, spec.getParserLocation(), VF.bool(spec.getAllowAmbiguity()), VF.integer(MAX_AMB_DEPTH),
```

Comment:
the `ParserSpecification` should get a new flag (also documented in the json side) to enable error recovery, and be able to pass in these flags, instead of getting these defaults.